### PR TITLE
adding about me section

### PR DESCRIPTION
**Pull request by ChrisJo93**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
-// import Link from "next/link"
 import React from "react"
-// import NavBar from "../components/NavBar"
 import Header from "@/components/Header"
+import About from "@/components/About"
+import Banner from "@/components/Banner"
```
Comment (ChrisJo93) [inline src/app/page.tsx:4]: Added about and banner components just to make it cleaner to read. We aren't "reusing" these components but it makes the landing page code far more readable.

File: src/components/About.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div
+      id="about_us"
+      className="grid align-center justify-center bg-cover bg-center bg-no-repeat min-h-screen min-w-full bg-white text-slate-700 text-3xl"
+    >
+      <div className="text-center	leading-normal max-w-auto ">
+        <p className="justify-self-center mt-32 text-4xl mb-3">About Us</p>
+        <p className="mx-52 text-xl text-balance">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Dui sapien
```
Comment (ChrisJo93) [inline src/components/About.tsx:13]: Just some lorem ipsum. Thomas referred to this as "latin" in one of his old PRs.

File: src/components/Header.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from "react"
 const Header = () => {
   return (
-    <div className="h-20 grid gap-4 grid-cols-2 bg-rose-100 text-slate-700">
```
Comment (ChrisJo93) [inline src/components/Header.tsx:4]: This isn't grade school or our first html project. We don't need a rose pink header

Comment (TJGit11) [reply src/components/Header.tsx:4]: Definitely testing a color, chill

Comment (ChrisJo93) [reply src/components/Header.tsx:4]: lmao I was the one who did the pink

Comment (TJGit11) [reply src/components/Header.tsx:4]: The point stands even taller!

Comment (ChrisJo93) [reply src/components/Header.tsx:4]: I was making fun of myself with this one. I liked the pink but it doesn't scream professional

**Pull request merged**